tools/gce-change-disktype/README.md

```diff
@@ -0,0 +1,45 @@
+# Update GCE Instance disk-type
+
+Update disks attached to a GCE instance.
+
+## Script Process
+
+1. Stop the VM instance
+1. Get a list of the disks attached to the VM instance
+1. For each disk attached to the instance
+    1. Detach the disk from the VM instance
+    1. Snapshot the disk
```

Comment:
I think there is a step after this that is missing. After snapshotting, a new disk is created.

Reply:
Fixed.